test/fixtures/academic_calendars.yml

```diff
@@ -0,0 +1,20 @@
+one:
+  id: 1
+  name: 2017-2018 EĞİTİM ÖĞRETİM YILI AKADEMİK TAKVİMİ
+  year: 2017
+  academic_term_id: 4
```

Comment:
fixturelarda id belirtlemeyeceğin için böyle foreign_key'de tanımlamamalısın. rails o kadar akıllı ki, direk ilişkili fixture'un adını yazsan anlıyor. mesela burda:

```
academic_term :one
```

demen yeterli.